lib/src/components/NxAccordion/stateful/__tests__/NxStatefulAccordion.test.tsx

```diff
@@ -4,46 +4,213 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import { act } from 'react-dom/test-utils';
+import React from 'react';
+import userEvent from '@testing-library/user-event';
 
-import NxStatefulAccordion, { Props } from '../NxStatefulAccordion';
+import NxStatefulAccordion from '../NxStatefulAccordion';
 import NxAccordion from '../../NxAccordion';
-import * as enzymeUtils from '../../../../__testutils__/enzymeUtils';
+import { rtlRender, rtlRenderElement } from '../../../../__testutils__/rtlUtils';
+import { render } from '@testing-library/react';
+import NxButton from '../../../NxButton/NxButton';
 
 describe('NxStatefulAccordion', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxStatefulAccordion, {});
+
+  const quickRender = rtlRender(NxStatefulAccordion, {});
+  const renderEl = rtlRenderElement(NxStatefulAccordion, {});
 
   it('renders an NxAccordion with the specified props', function() {
     const onToggle = jest.fn(),
-        component = getShallowComponent({ onToggle, className: 'foo', id: 'bar' });
+        component = renderEl({ onToggle, className: 'foo', id: 'bar' });
```

Comment:
This classname isn't being tested

Reply:
> This classname isn't being tested. That functionality is tested in the next test after this one, so the `className` here can be removed.

Removed here: 6078c7cc795259938902c153bc24d6657a67e910